tests/utils/payload.test.ts

```diff
@@ -0,0 +1,405 @@
+import { Buffer } from "buffer/";
```

Comment:
Let's adjust these tests to test the correct TyepScript types for their input. Check that you can create payload, check that you can get the payload, make sure the return type is the same as the input type.